tamr_unify_client/models/project/mastering.py

```diff
@@ -76,4 +76,19 @@ def published_clusters(self):
         alias = self.api_path + "/publishedClusters"
         return Dataset(self.client, None, alias)
 
+    def record_clusters_with_data(self):
+        """Record clusters with data.
+
+        Call :func:`~tamr_unify_client.models.dataset.resource.Dataset.refresh` from
+        this dataset to generate clusters with data.
+
+        Function is a workaround because API is broken.
+
+        :returns: The record clusters with data represented as a dataset
+        :rtype: :class:`~tamr_unify_client.models.dataset.resource.Dataset`
+        """
+        unified_dataset = self.unified_dataset()
+        name = unified_dataset.name + "_dedup_clusters_with_data"
```

Comment:
you could instead add a comment in the code here specifying what the workaround is:

```suggestion
        # TODO(Nelson-Andrade) Replace this workaround with a direct API call once the API
        # has an alias for the "clusters with data" dataset. Until then, we grab the dataset by
        # constructing its name from the corresponding Unified Dataset's name
        name = unified_dataset.name + "_dedup_clusters_with_data"
```